packages/tasit-sdk/src/helpers/DecentralandUtils.js

```diff
@@ -188,6 +197,14 @@ export default class DecentralandUtils {
       ...filter,
       fromBlock,
     });
-    return logs.map(log => iface.parseLog(log));
+
+    const parsedLogs = logs.map(log => {
+      const { transactionHash } = log;
+      const parsedLog = iface.parseLog(log);
```

Comment:
What is iface again?

Reply:
Its an `ethers.js` helpers class for dealing with ABI, in that case it's transforming a `{ 0, 1, 2 }` transfer event object to `{ from, to, assetId }`.

Reply:
Ah ok, cool!

Reply:
```suggestion
      // ethers.js helpers class for dealing with ABI
      // transforming a { 0, 1, 2 } transfer event object to { from, to, assetId }
      const parsedLog = iface.parseLog(log);
```

Reply:
Done.

Reply:
Thanks! Resolving